types/angular-promise-tracker/angular-promise-tracker-tests.ts

```diff
@@ -1,5 +1,5 @@
 angular.module('promise-tracker-tests', []).run(['$q', 'promiseTracker',
```

Comment:
```suggestion
angular.module('promise-tracker-tests', []).run(['$q', '$http', 'promiseTracker',
```

Reply:
It's not absolutely necessary as the code is not run, but I would add `'$http'` to the injection tokens array.

Reply:
Oh, I didn't notice the array.